src/emqx_cm.erl

```diff
@@ -493,3 +542,19 @@ get_chann_conn_mod(ClientId, ChanPid) when node(ChanPid) == node() ->
 get_chann_conn_mod(ClientId, ChanPid) ->
     rpc_call(node(ChanPid), get_chann_conn_mod, [ClientId, ChanPid], ?T_GET_INFO).
 
+increment_connected_client_count() ->
+    cast({connected_client_count, inc}).
+
+decrement_connected_client_count() ->
+    cast({connected_client_count, dec}).
+
+get_connected_client_count() ->
```

Comment:
Here we introduce a synchronous call from stats collector to the channel manger process. Wouldn't be it a bit "safer" to use a general scheme, when `emqx_cm` updates an ets (although there is a couple of values), and `get_connected_client_count` reads from it?

I am not sure about this.

Reply:
good point.
if we use a public ets for the counters, increment and decrement can be done without the cast.
and no need for this call either, which can potentially block for a long time when `?CM` is busy

Reply:
or maybe this: https://www.erlang.org/doc/man/counters.html

Reply:
Sounds like creating a counter during `emqx_cm` initialization like `emqx_metrics` and using it to keep track of live connections is a better idea indeed than the casts/calls. I'll make the changes to use that. Thanks! :beers: